flake.nix

```diff
@@ -106,11 +106,45 @@
               dhall
```

Comment:
Also it would be nice if we could just get rid of this in this PR, too. I forgot to remove the Dhall CLI back when I switched configuration formats 

```suggestion

```